cmake/linker_script/common/common-rom.cmake

```diff
@@ -202,6 +202,10 @@ if (CONFIG_LOG)
   zephyr_iterable_section(NAME log_backend KVMA RAM_REGION GROUP RODATA_REGION SUBALIGN 4)
 endif()
 
+if (CONFIG_MULTI_LEVEL_INTERRUPTS)
+  zephyr_iterable_section(NAME intc_table KVMA RAM_REGION GROUP RODATA_REGION SUBALIGN 4)
```

Comment:
I wonder if it would make sense to have 3 iterable sections - one for each level 🤔

Reply:
One of the earlier version has that, with different tables for each level we can remove `level` from the `struct _irq_parent_entry` but we need multiple `STRUCT_SECTION_FOREACH_ALTERNATE`s for each of the tables in `multilevel_irq.c`

Reply:
> we need multiple `STRUCT_SECTION_FOREACH_ALTERNATE`s for each of the tables in `multilevel_irq.c`

Alternatively, you could just create a wrapper (static inline function or macro) that would accept `level` as a parameter and iterate only over that particular table.

Reply:
@ycsin - should here be additional sections defined here? I can't remember what the outcome was.